reference-architecture/azure-ansible/azureexpand.json

```diff
@@ -0,0 +1,219 @@
+        "Standard_DS4",
+        "Standard_DS11",
+        "Standard_DS12",
+        "Standard_DS13",
+        "Standard_DS14",
+        "Standard_GS1",
+        "Standard_GS2",
+        "Standard_GS3",
+        "Standard_GS4",
+        "Standard_GS5"
+      ],
+      "metadata" : {
+        "description" : "The size of the each Node Virtual Machine."
+      }
+    },
+    "RHNUserName" : {
+      "type" : "string",
+      "minLength" : 1,
+      "metadata" : {
+        "description" : "Redhat Subscription User Name or Email Address"
+      }
+    },
+    "RHNPassword" : {
+      "type" : "securestring",
+      "metadata" : {
+        "description" : "Redhat Subscription Password"
+      }
+    },
+    "SubscriptionPoolId" : {
+      "type" : "string",
+      "minLength" : 1,
+      "metadata" : {
+        "description" : "Pool ID of the redhat subscritpion to use"
+      }
+    },
+    "sshPrivateData" : {
+      "type" : "string",
+      "metadata" : {
+        "description" : "SSH rsa private key file as a base64 string."
+      }
+    },
+    "StartingNodeNumber" : {
+      "type" : "int",
+      "defaultValue" : 4,
+      "minValue" : 1,
+      "maxValue" : 30,
+      "metadata" : {
+        "description" : "Number of Openshift Nodes to deploy (max 30)"
+      }
+    }
+  },
+  "variables" : {
+    "baseTemplateUrl" : "https://raw.githubusercontent.com/openshift/openshift-ansible-contrib/master/reference-architecture/azure-ansible/",
+    "baseVMachineTemplateUriInfranode" : "[concat(variables('baseTemplateUrl'), 'infranode.json')]",
+    "baseVMachineTemplateUriNode" : "[concat(variables('baseTemplateUrl'), 'node.json')]",
+    "baseVMachineTemplateUriMaster" : "[concat(variables('baseTemplateUrl'), 'master.json')]",
+    "location" : "[resourceGroup().location]",
+    "virtualNetworkName" : "openshiftVnet",
+    "addressPrefix" : "10.0.0.0/16",
+    "infranodesubnetName" : "infranodeSubnet",
+    "infranodesubnetPrefix" : "10.0.2.0/24",
+    "nodesubnetName" : "nodeSubnet",
+    "nodesubnetPrefix" : "10.0.1.0/24",
+    "mastersubnetName" : "masterSubnet",
+    "mastersubnetPrefix" : "10.0.0.0/24",
+    "apiVersion" : "2015-06-15",
+    "infranodeStorageName" : "[concat(uniqueString(resourceGroup().id), 'infranodesa')]",
+    "nodeStorageName" : "[concat(uniqueString(resourceGroup().id), 'nodesa')]",
+    "masterStorageName" : "[concat(uniqueString(resourceGroup().id), 'mastersa')]",
+    "vhdStorageType" : "Standard_LRS",
+    "vnetId" : "[resourceId('Microsoft.Network/virtualNetworks', variables('virtualNetworkName'))]",
+    "infranodeSubnetRef" : "[concat(variables('vnetId'), '/subnets/', variables('infranodesubnetName'))]",
+    "nodeSubnetRef" : "[concat(variables('vnetId'), '/subnets/', variables('nodesubnetName'))]",
+    "masterSubnetRef" : "[concat(variables('vnetId'), '/subnets/', variables('mastersubnetName'))]",
+    "rhel" : {
+      "publisher" : "RedHat",
+      "offer" : "RHEL",
+      "sku" : "7.2",
```

Comment:
If worried about ocp 3.3 on rhel we shipped it about a month ago on aws with no issues